### ISSUE TITLE
boardswarm-cli ui mode could be more obvious after changing device mode

### ISSUE THREAD

**Issue by obbardc** [boardswarm/boardswarm#14]
When scrolling in UI mode (with `^a k`, `^a j`), a small green area is shown with the line number.

I wonder if it'd be good to show a general status bar, with some user feedback. An immediate one I can think of is showing state e.g. `Changed device mode to X` when changing device state.

**Comment by obbardc** [comment]
An option could be to rework the tui using something like https://github.com/ratatui-org/ratatui

**Comment by sjoerdsimons** [comment]
> An option could be to rework the tui using something like https://github.com/ratatui-org/ratatui

Fwiw the UI already uses tui from which ratatui was forked ;) we for sure should move to ratatui indeed but that doesn't impact your other requests

**Comment by obbardc** [comment]
Yeah it wasn't clear to me that ratatui was a continuation project rather than a full rewrite until you mentioned it :-).
I moved that forward in #16 anyway, but for sure adding a status bar and other UI bits isn't blocked on that.
